mr/src/test/java/org/apache/iceberg/mr/TestCatalogs.java

```diff
@@ -176,8 +178,10 @@ public void testCreateDropTableToCatalog() throws IOException {
     HadoopCatalog catalog = new CustomHadoopCatalog(conf, warehouseLocation);
     Table table = catalog.loadTable(identifier);
 
-    Assert.assertEquals(SchemaParser.toJson(SCHEMA), SchemaParser.toJson(table.schema()));
-    Assert.assertEquals(PartitionSpecParser.toJson(SPEC), PartitionSpecParser.toJson(table.spec()));
+    Assertions.assertThat(SchemaParser.toJson(table.schema()))
+        .isEqualTo(SchemaParser.toJson(SCHEMA));
+    Assertions.assertThat(PartitionSpecParser.toJson(table.spec()))
```

Comment:
please use the statically imported `assertThat()` method here and everywhere else

Reply:
Got it. Fixed this by doing static import in all changed files.